### ISSUE TITLE
Select Fields do not correctly select non-string values

### ISSUE THREAD

**Issue by rk** [heartcombo/simple_form#124]
I'm using Rails 3.0.3 with simple_form 1.2.2.

I have a binary attribute I display as a select box (another ticket for why I did that) that when I edit my record does not remember the value it was set to.

```
<%= f.input :test_boolean, :include_blank => false %>
```

You can save it, but the value isn't coerced to a string before checking to see whether the values match when outputting the select's options.  (I reported something similar for the official Rails form_for select implementation.

In other words, when testing to see if a select > option element is the selected value they are of different types when editing. A boolean is either a boolean value or an integer, that is compared against the string representation.
